staticfiles/js/requester/controllers/requester.controller.js

```diff
@@ -0,0 +1,80 @@
+/**
+* Requester Task Portfolio Controller
+* @namespace crowdsource.requester.controllers
+*/
+(function () {
+  'use strict';
+
+  angular
+    .module('crowdsource.requester.controllers')
+    .controller('RequesterTaskPortfolioController', RequesterTaskPortfolioController);
+
+  RequesterTaskPortfolioController.$inject = ['$location', '$scope', 'Authentication'];
+
+  /**
+  * @namespace HomeController
+  */
+  function RequesterTaskPortfolioController($location, $scope, Authentication) {
+    var vm = this;
+
+    $scope.user = {
+      TaskCategories : [{
```

Comment:
The backend api for this hasn't been made yet(totally). Once that is done , i will add it surely. But, till then shall this be merged so that i can clone the updated repo ?